app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -125,7 +124,6 @@ class AamvaStateMaintenanceWindow
       ],
       'NV' => [
         # Tuesdays to Sundays: 2:00 am. to 3:15 am. ET
-        # MW FIXME: This wraps around, does Tue-Sun get parsed correctly?
```

Comment:
For the record, I had tested both of these, but neglected to remove the comments.